File: api/src/api/schemas/rent.js
export default {
  type: "object",
  properties: {
    userId: { type: "integer" },
    bookId: { type: "integer" },
    penalty: { type: "number" },
    rentDate: { type: "string", format: "date-time" },
    devolutionDate: { type: "string", format: "date-time" },
    createDate: { type: "string", format: "date-time" },
    deletedDate: { type: "string", format: "date-time" },
  },
  required: ["userId", "bookId", "rentDate", "createDate"],
  additionalProperties: false,
};

File: api/BibliotecaSuindara/Data/Entities/Rent.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BibliotecaSuindara.Data.Entities
{
    public class Rent
    {
        [Key]
        public int Id { get; set; }
        [ForeignKey("Book")]
        public int BookId { get; set; }

        [ForeignKey("User")]
        public int UserId { get; set; }

        public double Penalty { get; set; }

        public DateTime RentDate { get; set; }
        public DateTime? DevolutionDate { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime? DeletedDate { get; set; }
    }
}
